#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_Makefile.dpatch by  <rousseau@imac.maison.bogus>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: do not include non-existing Makefile.inc

@DPATCH@
diff -urNad asedriveiiie-3.5~/asedriveiiieserial/Makefile asedriveiiie-3.5/asedriveiiieserial/Makefile
--- asedriveiiie-3.5~/asedriveiiieserial/Makefile	2007-02-21 09:27:37.000000000 +0100
+++ asedriveiiie-3.5/asedriveiiieserial/Makefile	2008-01-05 17:48:24.000000000 +0100
@@ -1,5 +1,5 @@
 
-include Makefile.inc
+#include Makefile.inc
 
 DRIVER_DIR=${DESTDIR}/${USBDROPDIR}/ifd-ASEDriveIIIe-Serial
 
diff -urNad asedriveiiie-3.5~/asedriveiiieusb/Makefile asedriveiiie-3.5/asedriveiiieusb/Makefile
--- asedriveiiie-3.5~/asedriveiiieusb/Makefile	2007-02-21 09:32:13.000000000 +0100
+++ asedriveiiie-3.5/asedriveiiieusb/Makefile	2008-01-05 17:48:34.000000000 +0100
@@ -1,5 +1,5 @@
 
-include Makefile.inc
+#include Makefile.inc
 
 DRIVER_DIR=${DESTDIR}/${USBDROPDIR}/ifd-ASEDriveIIIe-USB.bundle
 
